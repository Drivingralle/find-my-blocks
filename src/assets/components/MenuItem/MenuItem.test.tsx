import React from "react";
import renderer from "react-test-renderer";
import { MenuItem } from ".";

it("renders the MenuItem", () => {
  const tree = renderer.create(
    <MenuItem title="test-content" count={3} />).toJSON;
  expect(tree).toMatchSnapshot();
});

it("renders posts vs post based on count", () => {
  const posts = renderer.create(
    <MenuItem title="test-content" count={3} />
  ).toJSON;
  const post = renderer.create(
    <MenuItem title="test-content" count={1} />
  ).toJSON;
  expect(posts).toMatchSnapshot();
  expect(post).toMatchSnapshot();
});
