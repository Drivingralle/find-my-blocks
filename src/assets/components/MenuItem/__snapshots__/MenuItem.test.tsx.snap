// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders posts vs post based on count 1`] = `[Function]`;

exports[`renders posts vs post based on count 2`] = `[Function]`;

exports[`renders the MenuItem 1`] = `[Function]`;
